3.6results

```diff
@@ -0,0 +1,860 @@
+lib2to3.pgen2.tokenize: 'ISNONTERMINAL' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'ISTERMINAL' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'LBRACE' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'LEFTSHIFT' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'LEFTSHIFTEQUAL' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'LESS' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'LESSEQUAL' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'LPAR' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'LSQB' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'MINEQUAL' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'MINUS' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'NAME' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'NEWLINE' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'NOTEQUAL' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'NT_OFFSET' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'NUMBER' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'N_TOKENS' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'OP' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'PERCENT' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'PERCENTEQUAL' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'PLUS' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'PLUSEQUAL' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'RARROW' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'RBRACE' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'RIGHTSHIFT' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'RIGHTSHIFTEQUAL' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'RPAR' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'RSQB' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'SEMI' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'SLASH' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'SLASHEQUAL' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'STAR' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'STAREQUAL' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'STRING' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'TILDE' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'VBAR' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'VBAREQUAL' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'generate_tokens' is in __all__ but not in stub
+lib2to3.pgen2.tokenize: 'tok_name' is in __all__ but not in stub
+logging: 'BufferingFormatter' is in __all__ but not in stub
+logging: 'FATAL' is in __all__ but not in stub
+logging: 'fatal' is in __all__ but not in stub
+logging: 'raiseExceptions' is in __all__ but not in stub
+logging.config: 'RawConfigParser' is in stub but is not defined at runtime
+logging.handlers: 'SocketKind' is in stub but is not defined at runtime
+xml: 'dom' is in __all__ but not in stub
```

Comment:
These are in `__all__`, but they don't actually exist if you `import xml`. I think stubcheck should catch this case and ignore the names.